application/ui.extkeywidget.class.inc.php

```diff
@@ -211,14 +211,23 @@ public function DisplaySelect(WebPage $oPage, $iMaxComboLength, $bAllowTargetCre
 			$sClassAllowed = $oAllowedValues->GetClass();
 			$bAddingValue = false;
 
+			// #4792 - load only the required fields
```

Comment:
```suggestion
			// N°4792 - load only the required fields
```

Just change this as #XXX are usually used for SourceForge ticket reference.